docs/source/how_to_guides/inference/how_to_do_bootstrap_inference.ipynb

```diff
@@ -0,0 +1,1038 @@
+    "\n",
+    "In this example, we will work with the \"exercise\" example dataset taken from the seaborn library.\n",
+    "\n",
+    "The working example will be a linear regression to investigate the effects of exercise time on pulse.\n",
+    "\n",
+    "## Prepare the dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>Unnamed: 0</th>\n",
+       "      <th>id</th>\n",
+       "      <th>diet</th>\n",
+       "      <th>pulse</th>\n",
+       "      <th>time</th>\n",
+       "      <th>kind</th>\n",
+       "      <th>constant</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>0</td>\n",
+       "      <td>1</td>\n",
+       "      <td>low fat</td>\n",
+       "      <td>85</td>\n",
+       "      <td>1</td>\n",
+       "      <td>rest</td>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>1</td>\n",
+       "      <td>1</td>\n",
+       "      <td>low fat</td>\n",
+       "      <td>85</td>\n",
+       "      <td>15</td>\n",
+       "      <td>rest</td>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>2</td>\n",
+       "      <td>1</td>\n",
+       "      <td>low fat</td>\n",
+       "      <td>88</td>\n",
+       "      <td>30</td>\n",
+       "      <td>rest</td>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>3</td>\n",
+       "      <td>2</td>\n",
+       "      <td>low fat</td>\n",
+       "      <td>90</td>\n",
+       "      <td>1</td>\n",
+       "      <td>rest</td>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>4</td>\n",
+       "      <td>2</td>\n",
+       "      <td>low fat</td>\n",
+       "      <td>92</td>\n",
+       "      <td>15</td>\n",
+       "      <td>rest</td>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   Unnamed: 0  id     diet  pulse  time  kind  constant\n",
```

Comment:
When using ``sns.load_dataset("exercise", index_col=0)``  one could avoid the "Unnamed: 0" column, but that's just superficial.

Reply:
Thanks!